libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -137,22 +187,26 @@ void GLSLPostProcessor::spirvToToMsl(const SpirvBlob *spirv, std::string *outMsl
 
     auto executionModel = mslCompiler.get_execution_model();
 
-    auto duplicateResourceBinding = [executionModel, &mslCompiler](const auto& resource) {
+    // The index will be used to remap based on BindingIndexMap and
+    // the result becomes a [[buffer(index)]], [[texture(index)]] or [[sampler(index)]].
+    auto duplicateResourceBinding = [executionModel, &mslCompiler]
```

Comment:
Let's rename this lambda to something like `updateResourceBinding` since this no longer does a pure duplication.

Reply:
Ok It's updated.